src/elements/Burger/BurgerMenu.tsx

```diff
@@ -1,3 +1,7 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @next/next/no-html-link-for-pages */
+/* eslint-disable react/forbid-component-props */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
```

Comment:
`@typescript-eslint/no-unsafe-return` you shouldn't explicitly change the state in the render function, try to create a handler for this and change state inside the handler

`const handleBurgerClose = ...`
and use it without curring, `onClick={handleBurgerClose}`